Tetris/Labels.qml

```diff
@@ -0,0 +1,21 @@
+import QtQuick 2.0
+
+Item {
+    Text {
+        id: text_score
+        text: qsTr("SCORE:")
+        y: 80
+        x: 57
+        font.pixelSize: 30
```

Comment:
please use layouts with anchors to find the correct position instead of `x`, `y` properties such an approach will add flexibility in case if one of the elements is changed